Apps/Src/Amps.c

```diff
@@ -1,27 +1,43 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
 /** Amps.c
  * Wrapper that holds all electrical current related information of BeVolt's
  * battery pack.
  */
 
 #include "Amps.h"
 #include "AS8510.h"
+#include "os.h"
+#include "Tasks.h"
+#include "BSP_SPI.h"
+
+static OS_ERR err;
```

Comment:
Is there an advantage to keeping this global rather than local? Seems like just another thing that we have to use locks to access...

Reply:
I will change this